test/dynamo/test_fx_graph_runnable.py

```diff
@@ -136,7 +136,7 @@ def _exec_and_verify_payload(self):
             tmp.write(payload)
             tmp.flush()
             res = subprocess.run(
-                [sys.executable, tmp.name], capture_output=True, text=True, timeout=30
+                [sys.executable, tmp.name], capture_output=True, text=True, timeout=120
```

Comment:
2 mins doesn't seem too bad - do we need to cc CI team though as a heads up?

Reply:
This might be because I'm running on debug cpython and it's slower than release version. I'll remove the timeout increase for now and see what happens on release once 3.14 CI is ready.